doc/classes/Node2D.xml

```diff
@@ -69,21 +70,21 @@
 			<return type="void" />
 			<param index="0" name="radians" type="float" />
 			<description>
-				Applies a rotation to the node, in radians, starting from its current rotation.
+				Rotates this node by the given angle [param radians], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
 			</description>
 		</method>
 		<method name="to_global" qualifiers="const">
 			<return type="Vector2" />
 			<param index="0" name="local_point" type="Vector2" />
 			<description>
-				Transforms the provided local position into a position in global coordinate space. The input is expected to be local relative to the [Node2D] it is called on. e.g. Applying this method to the positions of child nodes will correctly transform their positions into the global coordinate space, but applying it to a node's own position will give an incorrect result, as it will incorporate the node's own transformation into its global position.
+				Returns the [param local_point] converted from this node's local space to global space. This is the opposite of [method to_local].
 			</description>
 		</method>
 		<method name="to_local" qualifiers="const">
 			<return type="Vector2" />
 			<param index="0" name="global_point" type="Vector2" />
 			<description>
-				Transforms the provided global position into a position in local coordinate space. The output will be local relative to the [Node2D] it is called on. e.g. It is appropriate for determining the positions of child nodes, but it is not appropriate for determining its own position relative to its parent.
+				Returns the [param global_point] converted from global space to this node's local space. This is the opposite of [method to_global].
```

Comment:
This is for consistency with https://github.com/godotengine/godot/pull/87440
However, it's a pretty nasty loss of information here. The corresponding 3D methods are not as verbose as this. At the moment I don't know what to make of it.

Reply:
It would be enough if "local space" was explained somewhere (maybe taking the previous descriptions as inspiration).